Makefile.cli

```diff
@@ -30,6 +30,7 @@ cli-local-release: cli-clean
 				;; \
 			windows) \
 				ARCHS='arm64 amd64'; \
+				EXT='.exe' \
```

Comment:
nit: I would
```suggestion
				EXT='.exe'; \
```
just in case we add another line after this one at some point.

Reply:
Fixed! Thanks!